src/middleware/json-api/rails-params-serializer.js

```diff
@@ -4,11 +4,13 @@ module.exports = {
   name: 'rails-params-serializer',
   req: (payload) => {
     if (payload.req.method === 'GET') {
-      payload.req.paramsSerializer = function (params) {
-        return Qs.stringify(params, {
-          arrayFormat: 'brackets',
-          encodeValuesOnly: true
-        })
+      payload.req.paramsSerializer = {
```

Comment:
should we update the related tests?

Reply:
Not sure that is necessary. The tests already prove that this change works. When you only upgrade axios without updating the `paramsSerializer` you'll find many tests failing. (can try that by checking out [e47d61c](https://github.com/twg/devour/pull/243/commits/e47d61c4db1f04fd7f8d16180027352f5a146b90) and run the tests).